bin/i18n/sync-hourofcode-out

```diff
@@ -8,7 +9,7 @@ require 'tempfile'
 def push_out_translated_files
   rename_downloads_from_crowdin_code_to_locale
   copy_from_i18n_source_to_hoc
-  clean_crowdin_markdown_errors
+  restore_sanitized_headers
```

Comment:
In `sync-codeorg-out` we restore the strings in the i18n directory _then_ distribute. This seems to do the opposite. Any reason why?

Reply:
nope!

I just put this step in where the old "mess with the headers" code was; I don't think the exact order is super important.